test/Snowflake_Tests/src/Snowflake_Spec.enso

```diff
@@ -394,7 +394,7 @@ snowflake_specific_spec suite_builder default_connection db_name setup =
             table_name = Name_Generator.random_name "TimestampTZ"
             table = default_connection.get.create_table table_name [Column_Description.Value "A" (Value_Type.Date_Time with_timezone=True), Column_Description.Value "rowid" Value_Type.Integer] primary_key=[]
             table.should_succeed
-            Panic.with_finalizer default_connection.get.drop_table table.name <|
+            Panic.with_finalizer (default_connection.get.drop_table table.name) <|
```

Comment:
Does this not give a compile-time error, having too many arguments?

Reply:
No, such stuff is only currently resolved at runtime.

Once #9812 is implemented, we'd get a warning if static type analysis were enabled.